src/Signature/SignatureV4.php

```diff
@@ -82,7 +82,9 @@ public function presign(
         $expires,
         array $options = []
     ) {
-        $startTimestamp = isset($options['start_time']) ? $this->convertToTimestamp($options['start_time']) : time();
+        $relativeTimeBase = isset($options['relativeTimeBase']) ? $options['relativeTimeBase'] : time();
+        $expiresTimestamp = $this->convertToTimestamp($expires,$relativeTimeBase);
```

Comment:
Ok - will wait for you guys to merge that, then will merge those updates into this branch.